ui/imports/shared/controls/GetSyncCodeDesktopInstructions.qml

```diff
@@ -1,34 +1,58 @@
 import QtQuick
+import QtQuick.Layouts
 
 import StatusQ.Core.Theme
+import StatusQ.Core
 
 import shared.controls
 import shared.views
 
-Column {
+ColumnLayout {
     id: root
 
     property int purpose: SyncingCodeInstructions.Purpose.AppSync
     property int type: SyncingCodeInstructions.Type.QRCode
 
-    spacing: 4
+    QtObject {
+        id: d
 
-    DecoratedListItem {
-        order: "1."
-        text1: qsTr("Ensure both devices are on the same network")
+        readonly property int itemHeight: 32
     }
 
-    DecoratedListItem {
-        order: "2."
-        text1: {
+    spacing: Theme.padding
+
+    StatusBaseText {
+        Layout.preferredHeight: d.itemHeight
```

Comment:
it usually doesn't play well with dynamic font scaling, could we use Layout's top/bottom margins here?